packages/wonder-blocks-tooltip/components/tooltip.js

```diff
@@ -1,12 +1,87 @@
 // @flow
 import * as React from "react";
 
-type Props = {||};
+import {Text as WbText} from "@khanacademy/wonder-blocks-core";
```

Comment:
Please avoid the use of `WbText`.  Instead, use one of the typography components.

Reply:
This is specifically to avoid those. We don't want to apply any styling changes at all. This is to allow us to anchor on a bit of a string in the middle of a typography element. So we want to be as light as we can. I believe this is a valid use of `Text` here.

Reply:
I just had a look at `Text` and it's a `<span>` which is `inline` which is great.  🙂  There was some talk about making it a `div` b/c all of the typography elements are block-level elements.